examples/piecestore-client/rpc/client/main.go

```diff
@@ -32,7 +32,7 @@ func main() {
 		log.Fatalf("did not connect: %s", err)
 	}
 	defer conn.Close()
-	psClient := client.NewPSClient(conn)
```

Comment:
rename `NewPSClient` to ==> `NewClient` . When it's imported it'll be it'll be prefixed with the type of client it is.